src/transformState.js

```diff
@@ -5,7 +5,37 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type } = action;
```

Comment:
```suggestion
    const { type } = action;
```

add here all properties instead of destructuring assignment in each case